### PR TITLE
No_bug_assigned: Remove an extra 'a'

### PR DESCRIPTION
**Pull request by xenolinux**
OCP version: 4.6+
[Preview](https://deploy-preview-45405--osdocs.netlify.app/openshift-enterprise/latest/updating/updating-cluster-within-minor.html)
QE ack is not required

### CODE REVIEW

**Review by GroceryBoyJr — COMMENTED** [review]
/LGTM - please stand by for further review and merge.

File: updating/updating-cluster-within-minor.adoc [hunk 1]
```diff
@@ -6,7 +6,7 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can update, or upgrade, an {product-title} cluster by using the web console. The following steps are updating a cluster within a minor a version. You can use the same instructions for updating a cluster between minor versions.
+You can update, or upgrade, an {product-title} cluster by using the web console. The following steps are updating a cluster within a minor version. You can use the same instructions for updating a cluster between minor versions.
```
Comment (nalhadef) [inline updating/updating-cluster-within-minor.adoc:9]: Suggestion: Change "...steps are updating a cluster..." to "...steps update a cluster..."

**Comment by nalhadef** [comment]
One small nit. Otherwise marking as ...

File: updating/updating-cluster-within-minor.adoc (hunk 1 above)
Comment (xenolinux) [reply updating/updating-cluster-within-minor.adoc]: Addressed

**Pull request merged**